app.js

```diff
@@ -58,8 +58,7 @@ app.use(function (state, emitter) {
   })
 
   emitter.on('map:zoom:add', function (x) {
-    state.map.setZoom(state.map.getZoom()+x)
-    state.map.draw()
+    emitter.emit('map:zoom:set', state.map.getZoom()+x)
   })
   emitter.on('map:zoom:set', function (x) {
     state.map.setZoom(x)
```

Comment:
We might want to do `emitter.emit('map:zoom:updated')` here and have a listener on that instead below?